go.mod

```diff
@@ -18,3 +20,5 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/go-rel/sql v0.15.1-0.20230926233117-4af81443c5e1 => ../sql/
```

Comment:
is this intentional or was forgotten to remove?

Reply:
It was forgotten to remove in one of previous PR

Reply:
@Fs02 removed